### DISCUSSION TITLE
psycopg2 version for postgres 13.1

### DISCUSSION THREAD

**Discussion by RahulMishravogo** [psycopg/psycopg2#1327] [Q&A]
Recently we decided to upgrade our postgres db from 9.6.11 to 13.2/13.1 but in the psycopg2 docs we can see that they have mentioned that psycopg2 implementation supports `PostgreSQL server versions from 7.4 to 13` so here i am confused like it will support 13.1 or 13.2 or not.

**Comment by dvarrazzo** [comment]
Yes it does

**Reply by RahulMishravogo** [reply]
thanks @dvarrazzo
